.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,48 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build wheels and sdist on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        python-version: ['3.7', '3.8', '3.9']
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Build wheels
+        run: |
+          pip install -U build
+          python -m build
+          
+      - uses: actions/upload-artifact@v2
+        with:
+          path: |
+            dist/*.whl
+            dist/*.tar.gz
+
+# Disable until PyPI integration.
+  # upload_pypi:
+  #   needs: [build]
+  #   runs-on: ${{ matrix.os }}
+  #   # upload to PyPI on every tag starting with 'v'
+  #   if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v')
```

Comment:
would we want to push to pypi on every push? i would think only on a merge or opening a pull request, right?

Reply:
I agree that wouldn't be good -- the second condition ensures that it checks for things prefixed with "v" like the a "v1.0" release (this was taken straight from the `cibuildwheel` repository: https://github.com/pypa/cibuildwheel/blob/c936190712ab49c83cf113afe3ca0f63f02aa9a4/examples/github-deploy.yml#L58).